ui/Screen/Wallet/Pool.svelte

```diff
@@ -1,17 +1,17 @@
 <script lang="ts">
-  import { Remote } from "../../DesignSystem/Component";
+  import { Remote } from "ui/DesignSystem/Component";
 
   import Outgoing from "./Pool/Outgoing.svelte";
   import Incoming from "./Pool/Incoming.svelte";
-  import ModalCollect from "../../Modal/Funding/Pool/Collect.svelte";
+  import ModalCollect from "ui/Modal/Funding/Pool/Collect.svelte";
 
-  import * as modal from "../../src/modal";
-  import * as fundingPool from "../../src/funding/pool";
-  import type { Pool } from "../../src/funding/pool";
-  import { TxKind, ongoing } from "../../src/transaction";
-  import { store as txs } from "../../src/transaction";
+  import * as modal from "ui/src/modal";
+  import * as fundingPool from "ui/src/funding/pool";
+  import type { Pool } from "ui/src/funding/pool";
+  import { TxKind, ongoing, store as txs } from "ui/src/transaction";
+  import { store } from "ui/src/wallet";
 
-  export let pool: Pool;
+  let pool: Pool;
```

Comment:
You can remove this line. `pool` will be defined below.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1900/commits/e4975da1e70e9305b7e16438684e13bef45ef15f